src/providers/DatasetBasketProvider.tsx

```diff
@@ -21,6 +28,21 @@ export const DatasetBasketProvider = ({
   children: ReactNode;
 }) => {
   const [basket, setBasket] = useState<Dataset[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      const basketInLocalStorage = localStorage.getItem("basket");
```

Comment:
question: can you please check if this is the best way to do that?

I mean, It may have data privacy implications, as we are storing selected datasets.

Reply:
@brunopacheco1 I mean that's how you would store datasets no? Other option would be to encrypt it but i don't see much value because to access localstorage, you need access to browser and if you have access to browser, you can also see them in UI